includes/class-wc-stripe-helper.php

```diff
@@ -722,6 +722,48 @@ public static function get_upe_settings_enabled_payment_method_ids( $gateway ) {
 		return $enabled_gateways;
 	}
 
+	/**
+	 * Reorders the list of available payment gateways in 'woocommerce_gateway_order' option to include the Stripe methods
+	 * in the order merchants have chosen in the settings.
+	 *
+	 * @param array $ordered_payment_method_ids Ordered Stripe payment method list.
+	 */
+	public static function add_stripe_methods_in_woocommerce_gateway_order( $ordered_payment_method_ids ) {
+		$gateway_order = get_option( 'woocommerce_gateway_order' );
+
+		$ordered_available_stripe_methods = [];
+		// Map the Stripe payment method list to the right format to save in the 'woocommerce_gateway_order' option.
+		foreach ( $ordered_payment_method_ids as $payment_method_id ) {
+			$gateway_id = str_starts_with( $payment_method_id, 'stripe' ) ? $payment_method_id : 'stripe_' . $payment_method_id;
```

Comment:
Just noticed this (I did the same on a PR of mine 😅 ). We should use an alternative for the `str_starts_with` function because it was introduced in [PHP 8.0](https://www.php.net/manual/en/function.str-starts-with.php#:~:text=str_starts_with-,(PHP%208),-str_starts_with%20%E2%80%94%20Checks), and we should support previous PHP versions.

Reply:
Thanks for raising this 🤦‍♀️ 
Fixed this in [bec337b](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3222/commits/bec337bcd4c032523d93d5545bc1de15e86a48ed)